process/peer/process.go

```diff
@@ -586,16 +589,22 @@ func (vs *validatorStatistics) verifySignaturesBelowSignedThreshold(
 	shardId uint32,
 	epoch uint32,
 ) error {
-
 	if epoch < vs.ratingEnableEpoch {
 		return nil
 	}
 
-	validatorAppereances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
-	computedThreshold := float32(validator.ValidatorSuccess) / float32(validatorAppereances)
+	validatorAppearances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
+	computedThreshold := float32(validator.ValidatorSuccess) / float32(validatorAppearances)
 
 	if computedThreshold <= signedThreshold {
-		newTempRating := vs.rater.RevertIncreaseValidator(shardId, validator.TempRating, validator.ValidatorFailure)
+		increasedRatingTimes := uint32(0)
+		if epoch < vs.belowSignedThresholdEnableEpoch {
```

Comment:
This is ok as it is. In other cases (where current epoch was unavailable) we used the epoch change notifier mechanism.

Reply:
ok!